shell/platform/darwin/ios/framework/Source/FlutterEngineTest.mm

```diff
@@ -100,4 +102,27 @@ - (void)testRunningInitialRouteSendsNavigationMessage {
                                        message:encodedSetInitialRouteMethod]);
 }
 
+- (void)testPlatformViewsControllerRenderingMetalBackend {
+  FlutterEngine* engine = [[FlutterEngine alloc] init];
+
+  // Run with an initial route.
+  [engine runWithEntrypoint:FlutterDefaultDartEntrypoint initialRoute:@"test"];
```

Comment:
seems like the entrypoint/initialroute doesn't really matter? If so, you can just call `run`

Reply:
Yup, will change it.